familyGuy_practice_REV/js/code_rev.js

```diff
@@ -1,28 +1,48 @@
 'use strict';
-
-
-
-//show picture and description
-function toggleDisplay(){
-	let button = document.getElementsByTagName('h2');
-	let span1 = document.getElementById('span1');
-	let span2 = document.getElementById('span2');
-	let span3 = document.getElementById('span3');
+//function to show / hide peter griffin 
+function togglePeter(){
+	
+	
+    let peter = document.getElementById('peter');
+
+    if(peter.classList.contains('isHidden')){
```

Comment:
Also consider using `classList.toggle` which can toggle a classList based on a condition like in this example (https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Examples).

The only downside is that `classList.toggle` isn't supported by all browsers so it will have to be polyfilled.

What you have is perfectly fine, just another option that's out there